projects/packages/videopress/src/client/block-editor/blocks/video/components/player/index.native.tsx

```diff
@@ -44,6 +68,17 @@ export default function Player( {
 
 			{ ! isRequestingEmbedPreview && <SandBox html={ html } /> }
 			{ ! html && <Text>{ __( 'Loading…', 'jetpack-videopress-pkg' ) }</Text> }
+
+			{ isSelected && (
+				<BlockCaption
+					clientId={ clientId }
+					onFocus={ onFocusCaption }
+					isSelected={ isCaptionSelected }
```

Comment:
Seems we need to change `isCaptionSelected` to `false` when the block is unselected, otherwise, selecting the block again focuses the caption text input automatically. This behavior differs from other media blocks with the caption field.

https://user-images.githubusercontent.com/14905380/228571813-2d15d798-71ce-4aac-a371-dc5312af8843.MP4

**NOTE:** In the above video capture, the first block is a VideoPress block. The second is an Image block.